<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tke_form_fill_subs_narah</class>
 <widget class="QWidget" name="tke_form_fill_subs_narah">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Заповнення запиту по субсидіях</string>
  </property>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>340</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Закрити</string>
   </property>
   <property name="icon">
    <iconset resource="tke/tke_rc">
     <normaloff>:/pictures/pictures/button_cancel.png</normaloff>:/pictures/pictures/button_cancel.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_original_dbf_file_path">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_find_original_dbf_file">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1. Знайти dbf для заповнення</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>2. Заповнити dbf файл</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_fill_dbf_file">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Заповнити файл</string>
   </property>
   <property name="icon">
    <iconset resource="tke/tke_rc">
     <normaloff>:/pictures/pictures/korganizer.png</normaloff>:/pictures/pictures/korganizer.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>3. Перегляд заповненого документа</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_review_dbf_file">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Перегляд</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>4. Надіслати документ по пошті (поки не імплементовано)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_send_via_email">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>290</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Надіслати по е-пошті</string>
   </property>
   <property name="icon">
    <iconset resource="tke/tke_rc">
     <normaloff>:/pictures/pictures/kolab-mdk.png</normaloff>:/pictures/pictures/kolab-mdk.png</iconset>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_fill_dbf_result">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>441</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string>Результат:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>40</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Заповнений dbf</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_find_processed_dbf_file">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_processed_dbf_file_path">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="tke/tke_rc"/>
 </resources>
 <connections/>
</ui>
